codecept.conf.js

```diff
@@ -41,11 +41,13 @@ const addPages = (include) => {
 
 module.exports.config = {
   tests: `./${TestDir}/**/*_test.js`,
-  timeout: 10000,
```

Comment:
This doesn't actually seem to affect anything... I disabled with 0 and nothing waited forever...